src/main/java/com/ebay/sonar/test/github/TestGithub.java

```diff
@@ -4,6 +4,15 @@
 
 	public void testJava(){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AIndentationCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Method 'testJava' is not designed for extension - needs to be abstract, final or empty. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.design.DesignForExtensionCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) publicMethodCommentRequirement Required [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=pmd%3ACommentRequired)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AUndocumentedApi)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) '{' is not preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck)